repos/common/actors/augeasscanner/actor.py

```diff
@@ -0,0 +1,68 @@
+from leapp.actors import Actor
+from leapp.models import AugeasInput
+from leapp.models import AugeasOutput, LensesList, LensItem
+from leapp.tags import ScanTag
+
+import augeas
+import json
+
+import augeasprivate
+
+
+class AugeasScanner(Actor):
+    name = 'augeas_scanner'
+    description = 'Actor scans config files using Augeas'
+    consumes = (AugeasInput,)
+    produces = (AugeasOutput,)
+    tags = (ScanTag,)
+
+    def __init__(self, **kwargs):
+        super(AugeasScanner, self).__init__(**kwargs)
+
+        # path to folder with custom lenses
+        self.lenses_folder = 'files/lenses'
+
+        self.augeas_tree = augeas.Augeas()
+
+    def process(self):
+        self.log.info("Starting augeas scanner")
+
+        # for msg in self.consume(AugeasInput):
+        #     self.consumed_msg = msg
+        #     self.log.info("Starting to process...")
+        try:
+            self.consumed_msg = self.consume(AugeasInput).next()
+
+            metadata = augeasprivate.process_augeas_data(
+                self.consumed_msg, self.lenses_folder, self.augeas_tree
```

Comment:
since, I already said that the constructor usage doesn't make sense, you can use the lenses_folder, and augeas_tree parts directly here.